core/src/test/java/org/mule/processor/chain/DefaultMessageProcessorChainTestCase.java

```diff
@@ -253,13 +328,15 @@ public void testMixedMPChainWithNullReturn2() throws MuleException, Exception
     public void testMixedMPChainWithVoidlReturn2() throws MuleException, Exception
     {
         DefaultMessageProcessorChainBuilder builder = new DefaultMessageProcessorChainBuilder();
-        builder.chain(new AppendingInterceptingMP("1"), new AppendingMP("2"),
-            new ReturnVoidMPInterceptongMP(), new AppendingMP("3"), new AppendingInterceptingMP("4"),
-            new AppendingMP("5"));
-        assertEquals("0before12after1", builder.build()
-            .process(getTestEventUsingFlow("0"))
-            .getMessage()
-            .getPayload());
+        builder.chain(new AppendingInterceptingMP("1"), getAppendingMP("2"),
+                      new ReturnVoidMPInterceptongMP(), getAppendingMP("3"), new AppendingInterceptingMP("4"),
+                      getAppendingMP("5"));
```

Comment:
Could do but I'd prefer not to because I think it makes assertions harder to read without requiring concatenations and I think thats more important in this case.  Anything major that constants would add, because it's really not important if one test method uses "1" and another one uses "one".